addons/sale/sale.py

```diff
@@ -176,6 +176,20 @@ def _resolve_section_id_from_context(self, cr, uid, context=None):
                 return int(section_ids[0][0])
         return None
 
+    def _amount_total(self, cr, uid, context=None):
+        vals = {}
+        invoiced = 0.0
+        if context is None:
+            context = {}
+        order_id = context.get('active_id', False)
```

Comment:
`dict.get()` returns `None` as default, so specifying `False` instead is useless, because you will simply use the return value in a `if`...